src/clickBtn.js

```diff
@@ -53,7 +53,12 @@ async function clickBtn(button, options = []) {
       let { MessageButton, MessageActionRow } = require("discord.js");
 
       if (button.customId === "create_ticket") {
-        let ticketname = `ticket_${button.user.id}`;
+        
+        let split = '';
+        let usr = button.user.id.split(split);
+        for (var i = 0; i < usr.length; i++) usr[i] = usr[i].trim();
+
+        let ticketname = `ticket-${usr[0]}${usr[1]}${usr[2]}${usr[3]}`

```

Comment:
You could replace these 6 lines with this one, which improves readability a lot.
```js
let ticketname = button.user.id.slice(0, 4)
```